docs/DeveloperGuide.md

```diff
@@ -1,38 +1,682 @@
 # Developer Guide
```

Comment:
Overall, the implementation section describes the commands that interacts with TransactionList and Transaction. I feel that more complex features should be mentioned, specifically how some of the classes interact with each other to provide a 'value-add' to the user using it.